code/modules/surgery/organs/pain.dm

```diff
@@ -31,16 +33,20 @@
 
 // message is the custom message to be displayed
 mob/living/carbon/human/proc/custom_pain(message)
-	if(stat >= UNCONSCIOUS)
+	if(stat == DEAD || status_flags & FAKEDEATH)
+		return
+	if(stat == UNCONSCIOUS && paralysis)
 		return
-
 	if(NO_PAIN in dna.species.species_traits)
 		return
 	if(reagents.has_reagent("morphine"))
 		return
 	if(reagents.has_reagent("hydrocodone"))
 		return
 
+	if(sleeping)
+		SetSleeping(0)
+		to_chat(src, "<span class='userdanger'>Pain wakes you up!</span>")
```

Comment:
Broken bones an damaged organs now counter some antag tactics and knockouts with this.

Yeah, not a good idea.

Reply:
>Gets sleepypenned

>VIGOROUSLY STABS SELF WITH SCREWDRIVER TO STAY AWAKE

Reply:
Chemicals set you to Paralyse and not sleeping, which wouldnt get you to wake up.

The sleep verb is basically the only thing that directly sets Sleeping without going through paralyse.